backend/scripts/add_buildings.ts

```diff
@@ -0,0 +1,64 @@
+import { Apartment, ApartmentWithId } from '@common/types/db-types';
+import { db } from '../src/firebase-config';
+import buildingData from '../src/data/buildings.json';
+
+const buildingCollection = db.collection('buildings');
+
+type BuildingData = {
+  id: number;
+  name: string;
+  address: string;
+  landlordId: string | null;
+  numBaths: number;
+  numBeds: number;
+  photos: string;
+  area: string;
+};
+
+const getAreaType = (areaName: string): 'COLLEGETOWN' | 'WEST' | 'NORTH' | 'DOWNTOWN' | 'OTHER' => {
+  switch (areaName.toUpperCase()) {
+    case 'COLLEGETOWN':
+      return 'COLLEGETOWN';
+    case 'WEST':
+      return 'WEST';
+    case 'NORTH':
+      return 'NORTH';
+    case 'DOWNTOWN':
+      return 'DOWNTOWN';
+    default:
+      return 'OTHER';
+  }
+};
+
+const formatBuilding = ({
+  id,
+  name,
+  address,
+  landlordId,
+  numBaths,
```

Comment:
Can remove `numBaths`, `numBeds`, and `photos` for now since they are not being used